### PR TITLE
Haiku OS build

### PR DESCRIPTION
**Pull request by Limows**

### CODE REVIEW

File: 2048.c [hunk 1]
```diff
@@ -58,8 +58,7 @@ void drawBoard(uint8_t board[SIZE][SIZE]) {
 			if (board[x][y]!=0) {
 				char s[8];
 				snprintf(s,8,"%u",(uint32_t)1<<board[x][y]);
-				uint8_t t;
-				t=7-strlen(s);
+				uint t=7-strlen(s);
```
Comment (mevdschee) [inline 2048.c:61]: why is this change needed?

Comment (mevdschee) [reply 2048.c:61]: This seems to be a `-std=c90` error, not present while compiling with `-std=c99`

Comment (Limows) [reply 2048.c:61]: I changed this line, because it was at her that gcc2 swore at Haiku. In any case, I have already changed it back, otherwise the assembly under the FreeBSD breaks down. I will try to compile under the haiku with the c99 parameter

**Comment by mevdschee** [comment]
@Limows can you test with the latest version under Haiku OS?

**Comment by mevdschee** [comment]
I'll reopen when there is more information.
